class.jetpack.php

```diff
@@ -7304,7 +7304,7 @@ public static function load_jetpack_gutenberg() {
 		 *
 		 * @param bool true Whether to load Gutenberg blocks
 		 */
-		if ( ! Jetpack::is_gutenberg_available() || ! apply_filters( 'jetpack_gutenberg', true ) ) {
```

Comment:
Given that this is a Gutenberg-specific action, the `is_gutenberg_available` check is likely redundant.